src/test/config.sol

```diff
@@ -1789,5 +1789,35 @@ contract Config {
             calc_cut:     0,
             lerp:         false
         });
+        afterSpell.collaterals["RETH-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      5 * MILLION,
+            aL_gap:       3 * MILLION,
+            aL_ttl:       8 hours,
+            line:         0,
+            dust:         15 * THOUSAND,
+            pct:          150,
+            mat:          170_00,
+            liqType:      "clip",
+            liqOn:        true,
+            chop:         13_00,
+            cat_dunk:     0,
+            flip_beg:     0,
+            flip_ttl:     0,
+            flip_tau:     0,
+            flipper_mom:  0,
+            dog_hole:     2 * MILLION,
+            clip_buf:     110_00,
+            clip_tail:    120 minutes,
```

Comment:
Do we want this as 7_200 (or 7200) or as 120 minutes? Semantic choice here.

Reply:
To be consistent we should with the other collaterals I think we should leave this as minutes - but I am indifferent